### DISCUSSION TITLE
implement different bert model from hugging face

### DISCUSSION THREAD

**Discussion by us** [explosion/spaCy#10308] [Help: Coding & Implementations] [answered]
Currently `en_core_web_trf` uses `roberta-base` model by default, I want to use a different bert model in hugginface, how can I implement it, or how can I make it a package like `en_core_web_trf`?

Thank you!

**Comment by polm** [comment, marked as answer]
We just made [a new FAQ post](https://github.com/explosion/spaCy/discussions/10327) about this that should help you. If that doesn't answer your question let us know!
